en/ch02.md

```diff
@@ -41,7 +43,27 @@ Upon completion of the setup you will be prompted to the settings menu, but firs
 ![sync](media/daemon-launch.png)
 
 If you want to create a normal wallet using your personal full node, you don't need to do anything, let the countdown finish, then wait until your node is fully synced.  
-If you need some special settings, like setting up a view-only wallet or adding the blockchain manually, go to [section 3](#3-settings).  
+If you need some special settings, like setting up a view-only wallet or adding the blockchain manually, go to [Settings](#settings).  
+
+## Create new wallet from hardware
+
+### Create the wallet
+![2.4.1](media/create_hardware_wallet.png)
+
+(1) **Wallet name:** Give a name for your wallet (in this example `ledger-test` is used).  
+(2) **Wallet location:** Select the destination folder of the wallet.   
+(3) **Create a new wallet from device:** Select this option if this is the first time you use a hardware wallet.  
+(4) **Restore a wallet from device:** Choose this option if you already had a hardware wallet and want to restore it.  
+**Restore Height (optional):** *Restore only:* If you know the block height that your wallet was created at, you can specify it here so the wallet doesn't have to scan the entire blockchain looking for your funds. For example, if your first transaction was included in block `1350000`, you should put a slightly lower height (e.g. `1330000`) so the wallet will start scanning from there, saving you some time.  
+(5) **Subaddress lookahead (optional):** pregenerate a number of accounts with a number of subaddresses each  
+(6) **Device name:** Select the hardware wallet you want to use (at the moment, only 'Ledger' is available)  
```

Comment:
"at the moment, only 'Ledger' is available" -> Both Ledger and Trezor are available currently.

Reply:
Only on master, but i assume we could move forward and remove this statement as the guide will be embedded in the point release anyway. @erciccione ?

Reply:
> Only on master, but i assume we could move forward and remove this statement as the guide will be embedded in the point release anyway.

Yes, that's the correcct approach